### PR TITLE
Fix overflowing docs pages on narrow devices

### PR DESCRIPTION
**Pull request by georgegillams**
I've replaced `marked` with `react-markdown`, which is a react wrapper around `marked`.
This gives us more control over how MD is rendered on our docs and information pages, and saves us the headache of dangerously setting inner HTML.

Wherever we allow the renderer to do it's default things, our bareHtml metal styles are applied. So there should be no visual difference due to the swap 👍

### CODE REVIEW

File: packages/bpk-mixins/src/mixins/_typography.scss [hunk 1]
```diff
@@ -591,6 +591,7 @@
 
 @mixin bpk-table__cell {
   padding: $bpk-spacing-sm;
+  overflow: hidden;
```
Comment (georgegillams) [inline packages/bpk-mixins/src/mixins/_typography.scss]: obviously we would need to assess the blast radius of this change - as the mixin could be used in other components...

**Comment by shaundon** [comment]
@georgegillams is this still valid? Might be worth having a before/after video or gif to illustrate the change if so

**Comment by georgegillams** [comment]
https://github.com/Skyscanner/backpack/pull/1326#issuecomment-467446142
Yeah - sorry I'd kinda forgotten about this.
There are actually videos in the attached zip 👍

File: packages/bpk-mixins/src/mixins/_typography.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (georgegillams) [reply packages/bpk-mixins/src/mixins/_typography.scss]: That's a great shout!

File: UNRELEASED.md [hunk 2]
```diff
@@ -1,2 +1,6 @@
 # Unreleased
 
+**Fixed:**
+  - bpk-mixins:
+    - Table cell mixin will now prevent content from overflowing into the next cell in webkit browsers, providing scroll behaviour instead.
```
Comment (shaundon) [inline UNRELEASED.md]:
```suggestion
    - Table cell mixin will now prevent content from overflowing into the next cell in WebKit browsers, providing scroll behaviour instead.
```

File: packages/bpk-mixins/src/mixins/_typography.scss [hunk 3]
```diff
@@ -591,6 +591,11 @@
 
 @mixin bpk-table__cell {
   padding: $bpk-spacing-sm;
+  overflow: auto;
+
+  &::-webkit-scrollbar {
+    display: none;
```
Comment (shaundon) [inline packages/bpk-mixins/src/mixins/_typography.scss]: Why are we hiding the scrollbar?

Comment (georgegillams) [reply packages/bpk-mixins/src/mixins/_typography.scss]: Good question - probably one for @matthewdavidson...

Comment (georgegillams) [reply packages/bpk-mixins/src/mixins/_typography.scss]: It doesn't seem to actually have any effect...

Comment (shaundon) [reply packages/bpk-mixins/src/mixins/_typography.scss]: Did you try it in Safari?

Comment (georgegillams) [reply packages/bpk-mixins/src/mixins/_typography.scss]: Ah - only tried it on mobile safari before, and it makes no difference there.
On desktop it does, but tbh I'm not sure it's the right thing to do. Maybe we should just leave it be? Scrollbars in MacOS are hardly invasive...

Comment (shaundon) [reply packages/bpk-mixins/src/mixins/_typography.scss]: Yeah I think leave it in - I'm concerned it'll harm a11y to remove them.

Comment (matthewdavidson) [reply packages/bpk-mixins/src/mixins/_typography.scss]: Can we get design input into this please? We need to choose an option that is the best for all browsers / platforms. I.e. what does this look like on chrome for windows?

Comment (georgegillams) [reply packages/bpk-mixins/src/mixins/_typography.scss]: AFAI can tell the `webkit-scrollbar` selector has no effect in edge.
Hiding the scroll bars in Safari and Chrome seems a little pointless as they are so unobtrusive anyway (ie they are invisible unless actively scrolling)

Comment (matthewdavidson) [reply packages/bpk-mixins/src/mixins/_typography.scss]: > (ie they are invisible unless actively scrolling)

Have you checked this on windows?

Comment (shaundon) [reply packages/bpk-mixins/src/mixins/_typography.scss]: Yeah invisible scrollbars is a macOS / iOS thing afaik

Comment (georgegillams) [reply packages/bpk-mixins/src/mixins/_typography.scss]: tbh - the more I think about this the more I think that the whole table should be scrolling, not the individual cell's content
Then we don't have to screw with scrollbars as the user / system setting will be approriate

Comment (matthewdavidson) [reply packages/bpk-mixins/src/mixins/_typography.scss]: Perhaps we could use the mobile scroll container ^^

Comment (georgegillams) [reply packages/bpk-mixins/src/mixins/_typography.scss]: Agreed (this was just a really quick gif I made in dev tools 😉)

Comment (georgegillams) [reply packages/bpk-mixins/src/mixins/_typography.scss]: Dogfooding yo!
https://www.reddit.com/r/funny/comments/b0ksok/pure_betrayal/?utm_source=share&utm_medium=ios_app

**Comment by shaundon** [comment]
Happy to approve once conflicts are resolved.

**Pull request merged**